### DISCUSSION TITLE
grub4efi parttype command

### DISCUSSION THREAD

**Discussion by Taviruni** [chenall/grub4dos#327] [Q&A] [answered]
Nice you activated this section, it seems a good option to ask our questions.

I would like to have a few more info about the G4E command **parttype**

Only info I can get using help command is:   parttype [PART] [TYPE]

Is it only usefull to give us info of current partition attributes (type)?

or is it also capable to edit the selected partition attibutes?

Thanks in advance for your answer.

alacran (on reboot.pro)

**Comment by a1ive** [comment, marked as answer]
>parttype
>Display the current partition partition type
>parttype (hd0,0)
>Display the partition type of the first hard disk first partition
>parttype (hd0,0) 0x17
>The type of the first partition of the first hard disk partition to 0x17

https://rmprepusb.com/rmprep/grub4dos-commands-primer.html

**Reply by Taviruni** [reply]
Thanks for your quick answer.

So the command works same way in G4D and G4E.

alacran (on reboot.pro)

**Reply by Taviruni** [reply]
I would like to know also if **hide** and **unhide** commands (valid for G4D), also work in G4E.

**Reply by a1ive** [reply]
> I would like to know also if **hide** and **unhide** commands (valid for G4D), also work in G4E.

No.
